kivy/clock.py

```diff
@@ -408,7 +408,7 @@ def _get_sleep_obj():
 
         _kernel32 = ctypes.windll.kernel32
 
-        def _get_sleep_obj():
+        def _get_sleep_obj():  # noqa
```

Comment:
I would suggest to be more specific on the error/warning you want to ignore, flake8 supports that